src/app/(registration)/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -2,9 +2,28 @@
 "use client";
 
 import { SignIn } from "@clerk/nextjs";
+import { OakBox } from "@oaknational/oak-components";
 
 import { formAppearanceStyles } from "../../formAppearanceStyles";
+import { AuthLayout } from "../../auth-layout";
+
+import { getIllustrationAsset } from "@/image-data";
+import CMSImage from "@/components/SharedComponents/CMSImage";
 
 export default function SignInPage() {
-  return <SignIn appearance={formAppearanceStyles} />;
+  return (
+    <AuthLayout
+      asideSlot={
+        <OakBox $maxWidth="all-spacing-21">
+          <CMSImage
+            image={getIllustrationAsset("auth-acorn")}
+            $width="100%"
+            $objectFit="contain"
+          />
+        </OakBox>
+      }
+    >
```

Comment:
Sign-up and sign-in pages no longer have a common layout so it's now a regular component instead of an app layout